bftengine/src/bftengine/ReadOnlyReplica.hpp

```diff
@@ -0,0 +1,60 @@
+// Concord
+//
+// Copyright (c) 2018-2019 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").  You may not use this product except in
+// compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright notices and license terms. Your use of
+// these subcomponents is subject to the terms and conditions of the sub-component's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "ReplicaForStateTransfer.hpp"
+
+namespace bftEngine::impl {
+
+class PersistentStorage;
+/**
+ *
+ */
+class ReadOnlyReplica : public ReplicaForStateTransfer {
+ public:
+  ReadOnlyReplica(const ReplicaConfig&,
+                  IStateTransfer*,
+                  std::shared_ptr<MsgsCommunicator>,
+                  std::shared_ptr<PersistentStorage> persistentStorage,
```

Comment:
One of two: define all parameter names or remove all of them.

Reply:
I'll do it in the next commit